### PR TITLE
Issue #1 - Create project: add CRA, Redux (with env), react-router, r…

### PR DESCRIPTION
**Pull request by myroslav-kolomiiets**
…eselect, lodash/fp, flow, prettier

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-MIT License
-
```
Comment (zonzujiro) [inline LICENSE]: Why? :( You think we don't need license?

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
Comment (zonzujiro) [inline src/index.js]: Pls, create `redux-store` and integrate app with it

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import * as _ from 'lodash/fp';
```
Comment (zonzujiro) [inline src/App.js]: Unused variable.

Please, add `eslint` to project also

**Review by zonzujiro — CHANGES_REQUESTED** [review]
Pls, see comments.

File: src/index.js (hunk 2 above)
Comment (zonzujiro) [reply src/index.js]: Or you will do this in next PR?

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
```
Comment (zonzujiro) [inline src/App.js]: Flow will check this file without `Flow` comment? If no, pls add comment here and in `index.js` to

**Review by zonzujiro — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (myroslav-kolomiiets) [reply LICENSE]: Sorry, it was deleted by mistake. I returned it back in next commit.

File: src/index.js (hunk 2 above)
Comment (myroslav-kolomiiets) [reply src/index.js]: I am going to add it in next PR, if it is ok for you. Or it's better to do it in scope of this PR?

File: src/App.js (hunk 4 above)
Comment (myroslav-kolomiiets) [reply src/App.js]: Done.

File: src/App.js (hunk 3 above)
Comment (myroslav-kolomiiets) [reply src/App.js]: Done.

**Review by zonzujiro — APPROVED** [review]
Nice job! Squash and merge

**Comment by zonzujiro** [comment]
@iammiro pls, don't forget to delete brunch after merging :)

**Pull request merged**